UI/common/navbar

nakagame opened the pull request:
[UI/Common/Navbar] - Created Common Navbar 
- Added Style.css
   1. Style sheet color
   2. Secondary Font -> Montserrat
   3. Navbar icon size
- Modified layouts > app.blade.php
   1. Change main font -> Popins 
- Created Navbar.blade.php
   This file is in layouts > navbar.blade.php

nakagame committed:
[UI/Common/Navbar] - Update Navbar
- Delete nav section from app.blade.php
- Created a new file into layout > navbar.blade.php

nakagame committed:
Merge branch 'main' into UI/Common/Navbar

nakagame committed:
[UI/Common/Navbar] - Created Common Navbar
- Added style CSS
  -   add stylesheet color
  -   add second font -> montesrat
  -   add navbar icon size
- Modified app.layouts
  -   modified main font -> popins
- Cerate navbar.blade.php

Ditzler01 commented on resources/views/layouts/navbar.blade.php:
```diff
@@ -0,0 +1,143 @@
+<style>
```
Put the style to a different file. You can put it under public folder then css.

nakagame committed:
[UI/Common/Navbar] - Put the style to a public>css>navbar.css.
- Created a new file -> navbar.css
- Remove Style css from navbar.blade.php
- Move to the css to public>css>navbar.css.
